oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -69,45 +71,44 @@ if [ -n "$systemctl" ] && [ $(running_systemd) -eq 0 ]; then
-	rm -f /lib/systemd/system/mcs-exemgr.service
-        rm -f /usr/lib/systemd/system/mcs-primproc.service
-	rm -f /lib/systemd/system/mcs-primproc.service
-        rm -f /usr/lib/systemd/system/mcs-workernode.service
-	rm -f /lib/systemd/system/mcs-workernode.service
-        rm -f /usr/lib/systemd/system/mcs-writeengineserver.service
-	rm -f /lib/systemd/system/mcs-writeengineserver.service
-        rm -f /usr/lib/systemd/system/mcs-loadbrm.service
-	rm -f /lib/systemd/system/mcs-loadbrm.service
+    rm -f /usr/lib/systemd/system/mariadb-columnstore.service
+    rm -f /lib/systemd/system/mariadb-columnstore.service
+    rm -f /usr/lib/systemd/system/mcs-controllernode.service
+    rm -f /lib/systemd/system/mcs-controllernode.service
+    rm -f /usr/lib/systemd/system/mcs-ddlproc.service
+    rm -f /lib/systemd/system/mcs-ddlproc.service
+    rm -f /usr/lib/systemd/system/mcs-dmlproc.service
+    rm -f /lib/systemd/system/mcs-dmlproc.service
+    rm -f /usr/lib/systemd/system/mcs-exemgr.service
+    rm -f /lib/systemd/system/mcs-exemgr.service
+    rm -f /usr/lib/systemd/system/mcs-primproc.service
+    rm -f /lib/systemd/system/mcs-primproc.service
+    rm -f /usr/lib/systemd/system/mcs-workernode.service
+    rm -f /lib/systemd/system/mcs-workernode.service
+    rm -f /usr/lib/systemd/system/mcs-writeengineserver.service
+    rm -f /lib/systemd/system/mcs-writeengineserver.service
+    rm -f /usr/lib/systemd/system/mcs-loadbrm.service
+    rm -f /lib/systemd/system/mcs-loadbrm.service
     systemctl daemon-reload
 else
-	chkconfig=`which chkconfig 2>/dev/null`
-	if [ -n "$chkconfig" ]; then
-		chkconfig columnstore off > /dev/null 2>&1
-		chkconfig --del columnstore > /dev/null 2>&1
-		rm -f /etc/init.d/columnstore > /dev/null 2>&1
-	else
-		updaterc=`which update-rc.d 2>/dev/null`
-		if [ -n "$updaterc" ]; then
-			update-rc.d -f columnstore remove > /dev/null 2>&1
-			rm -f /etc/init.d/columnstore > /dev/null 2>&1	
-		fi
-	fi
+    chkconfig=`which chkconfig 2>/dev/null`
+    if [ -n "$chkconfig" ]; then
+        chkconfig columnstore off > /dev/null 2>&1
+        chkconfig --del columnstore > /dev/null 2>&1
+        rm -f /etc/init.d/columnstore > /dev/null 2>&1
+    else
+        updaterc=`which update-rc.d 2>/dev/null`
+        if [ -n "$updaterc" ]; then
+            update-rc.d -f columnstore remove > /dev/null 2>&1
+            rm -f /etc/init.d/columnstore > /dev/null 2>&1	
+        fi
+    fi
 fi
 
 #make copy of Columnstore.xml
 /bin/cp -f @ENGINE_SYSCONFDIR@/columnstore/Columnstore.xml @ENGINE_SYSCONFDIR@/columnstore/Columnstore.xml.rpmsave > /dev/null 2>&1
 /bin/cp -f @ENGINE_SYSCONFDIR@/columnstore/storagemanager.cnf @ENGINE_SYSCONFDIR@/columnstore/storagemanager.cnf.rpmsave > /dev/null 2>&1
 /bin/cp -f @MARIADB_MYCNFDIR@/columnstore.cnf @MARIADB_MYCNFDIR@/columnstore.cnf.rpmsave > /dev/null 2>&1
-cp @ENGINE_SUPPORTDIR@/myCnf-include-args.text @ENGINE_SUPPORTDIR@/myCnf-include-args.text.rpmsave >& /dev/null
```

Comment:
only file left in /usr/share/columnstore after package removal. I don't really think we need this if we do a removal.